pkgs/servers/monitoring/grafana/plugins/grafana-oncall-app/default.nix

```diff
@@ -3,8 +3,8 @@
 grafanaPlugin {
   pname = "grafana-oncall-app";
   versionPrefix = "v";
-  version = "1.8.5";
-  zipHash = "sha256-HuZYHPTWm0EPKQbmapALK2j+PzM+J7gcWM9w8vU2yI0=";
+  version = "v1.9.26";
+  zipHash = "sha256-SMXjyU3OCHfUVPrSbgKkpXF/Hk6J8LpUg+kjJZNc/bE=";
```

Comment:
Fails with

```
grafana-oncall-app-vv1.9.26.zip> trying https://grafana.com/api/plugins/grafana-oncall-app/versions/vv1.9.26/download
grafana-oncall-app-vv1.9.26.zip>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
grafana-oncall-app-vv1.9.26.zip>                                  Dload  Upload   Total   Spent    Left  Speed
grafana-oncall-app-vv1.9.26.zip>   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
grafana-oncall-app-vv1.9.26.zip> curl: (22) The requested URL returned error: 404
grafana-oncall-app-vv1.9.26.zip> error: cannot download grafana-oncall-app-vv1.9.26.zip from any mirror
```

I believe this is a bug in the update script?

Reply:
Yeah, the common package thingy does weird stuff with `versionPrefix` and I'm not sure if its the responsibility of the updated script or `update-package-version` to deal with it.

Regardless it seems like in the time between my update and your review the upstream has yet again dropped the `v` prefix so I don't think there is any action necessary besides just removing `versionPrefix` from the package.